rcl_lifecycle/test/test_rcl_lifecycle.cpp

```diff
@@ -337,7 +340,6 @@ TEST(TestRclLifecycle, state_machine) {
   ret = rcl_lifecycle_state_machine_fini(&state_machine, nullptr, &allocator);
   EXPECT_EQ(ret, RCL_RET_ERROR);
   rcutils_reset_error();
-  std::cout << "state_machine: " << __LINE__ << std::endl;
```

Comment:
Should this be part of another PR to be able of backporting removing that msg?

Reply:
Sounds good. Opened https://github.com/ros2/rcl/pull/773

Reply:
Rebased onto #773